File: Day6/Solution.cpp
void printEvenIndexChar(string s){
        int len = s.length();
        for (int i = 0;i < len; i++){
            if(i%2 ==0)
                cout<< s[i];
        }

}

void printOddIndexChar(string s){
       int len = s.length();
       for (int i = 0;i < len; i++){
           if(i%2 !=0)
               cout << s[i];
       }

    }

int main() {

    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    string s;
    int t;
    cin >> t;
    for (int a0 = 0; a0 < t; a0++){
        cin >> s;
        printEvenIndexChar(s);
        cout << " ";
        printOddIndexChar(s);
        cout << endl;
    }
    return 0;
}

File: Day6/Solution.py
#!/bin/python

import sys

def printEvenIndexChar(s):
    l = len(s)
    output = ""
    for i in range(0,l,2):
        output += s[i]
    return output

def printOddIndexChar(s):
    l = len(s)
    output = ""
    for i in range(1,l,2):
        output += s[i]
    return output

t = int(input())
for a0 in range(0,t):
    s = input()
    print(printEvenIndexChar(s) + " " + printOddIndexChar(s))
